archey/output.py

```diff
@@ -45,8 +45,10 @@ def __init__(self, **kwargs):
         if preferred_logo_style and hasattr(logo_module, f"LOGO_{preferred_logo_style}"):
             self._logo = getattr(logo_module, f"LOGO_{preferred_logo_style}").copy()
             self._colors = getattr(logo_module, f"COLORS_{preferred_logo_style}").copy()
+            self._logo_options = getattr(logo_module, f"OPTIONS_{preferred_logo_style}", {}).copy()
```

Comment:
I don't think we need this `preformat` flag and "options" notion here : if your logo don't contain any `c[ ]` values, it will be formatted "as-is" 👍

Reply:
This issue is with the .format(), since the string contains unmatched {}.